client/components/olark-chat/olarkErrorBoundary.jsx

```diff
@@ -0,0 +1,38 @@
+import config from '@automattic/calypso-config';
+import { Component } from 'react';
+import { logToLogstash } from 'calypso/lib/logstash';
+
+class OlarkErrorBoundary extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = { error: null, errorInfo: null };
+	}
```

Comment:
We could remove the `constructor()` method altogether and set state directly as instance property:

```suggestion
	state = { error: null, errorInfo: null };
```

Reply:
Fixed in [518e27c](https://github.com/Automattic/wp-calypso/pull/62483/commits/518e27c8587533f36306532feaf1c8c0bc558476).